contracts/SiphonDepositor.sol

```diff
@@ -0,0 +1,185 @@
+pragma solidity 0.8.11;
+
+import { IERC20 } from "@openzeppelin/contracts-0.8/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts-0.8/token/ERC20/utils/SafeERC20.sol";
+import { Ownable } from "@openzeppelin/contracts-0.8/access/Ownable.sol";
+import { AuraMath } from "./AuraMath.sol";
+import { IAuraLocker } from "./Interfaces.sol";
+
+// prettier-ignore
+interface IDeposit{
```

Comment:
Lets clean up all these interfaces and get them out of here

Reply:
Donzo